modules/tripleliftBidAdapter.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+```
+Module Name:  Triplelift Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: bzellman@triplelift.com
+```
+
+# Description
+
+Connects to Triplelift Exchange for bids.
+Triplelift bid adapter supports Banner format only.
+
+# Test Parameters
+```
+var adUnits = [{
+    code: 'banner-div',
+    mediaTypes: {
+        banner: {
+            sizes: [[300, 600], [300, 250], [320, 90]],
+        }
+    },
+    bids: [
+    {
+        bidder: 'triplelift',
+        params: {
+           inventoryCode: 'forbes_main',
+           floor: 1.009
+        }
+    },{
+        bidder: 'appnexus',
```

Comment:
any reason you included an AppNexus bid in your test?

Reply:
wanted to see in prebid debug that I was seeing multiple bidders, would you like me to remove?